src/Reviews/Reviews.jsx

```diff
@@ -9,150 +9,174 @@ class Reviews extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      productID: '19092',
-      reviews: [],
+      productID: '19093',
```

Comment:
Do we still need to have an initial productID hardcoded in? 

If possible, would it make sense to have this state initialized as '', and have error handling such that if blank it doesn't fire API requests? Then update it when the product id is passed down from parent props?

If this is just an intermediate state, cool, otherwise we may want to fix this.

Reply:
i'll take you up on this suggestion